mlflow/protos/service.proto

```diff
@@ -707,6 +719,42 @@ message ListExperiments {
   }
 }
 
+message SearchExperiments {
+  option (scalapb.message).extends = "com.databricks.rpc.RPC[$this.Response]";
+  option (scalapb.message).extends = "com.databricks.mlflow.api.MlflowTrackingMessage";
```

Comment:
Can we remove this line? Only needed on databricks

```suggestion
```

Reply:
Thanks for the catch!